data/iKunal-Singh/iKunal-Singh.json

```
@@ -0,0 +1,101 @@
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/_ikunalsingh" },
+    { "icon": "FaGithub", "url": "https://github.com/iKunal-Singh" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/ikunalsingha" },
+    { "icon": "FaSoundCloud", "url": "https://soundcloud.com/kunal-singh-558758897" },
+  ],
+  "testimonials": [ ],
+  "links": [
+    {
+      "name": "Follow for daily Lit tweets",
+      "url": "https://twitter.com/_ikunalsingh",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow my DevOps journey",
+      "url": "https://github.com/iKunal-Singh",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ikunalsingha",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Check Out SoundCloud",
+      "url": "https://soundcloud.com/kunal-singh-558758897",
+      "icon": "FaSoundCloud"
+    },
+    {
+      "name": "Open Source, DevOps and Linux Blog",
+      "url": "https://ikunalsingh.hashnode.dev/",
+      "icon": "SiHashnode"
+    },
+  ],
+  "milestones": [
+    {
+      "title": "The Big Drop",
+      "date": "2019",
+      "icon": "SiGooglescholar",
+      "color": "red",
+      "description": "I took a drop year to focus on preparing for the JEE Mains exam and was able to achieve my goal by scoring **98.711 percentile** in the test.",
+      "url": "https://github.com/Pradumnasaraf"
+    },
+    {
+      "title": "The Beginning",
+      "date": "2019",
+      "icon": "FaHourglassStart",
+      "color": "blue",
+      "description": "Started my journey as a developer",
+    },
+    {
+      "title": "First Hacktoberfest",
+      "date": "2019",
+      "icon": "FaGlobeAsia",
+      "color": "purple",
+      "description": "Participated in Hacktoberfest",
+    },
+    {
+      "title": "First Hackathon",
+      "date": "2020",
+      "icon": "FaGlobe",
+      "color": "black",
+      "description": "Participated in my first hackathon",
+    },
+    {
+      "title": "Started Music Production",
+      "date": "2020",
+      "icon": "SiSoundcloud",
+      "color": "orange",
+      "description": "Started producing music",
+    },
+    {
+      "title": "Started Youtube",
+      "date": "2020",
+      "icon": "SiYoutube",
+      "color": "red",
+      "description": "Started uploading my music on youtube and got more than **1 lakh impressions, 20K+ views and 50K+ watch time**",
+    },
+    {
+      "title": "Started Freelancing",
+      "date": "2022",
+      "icon": "FaDollarSign",
+      "color": "yellow",
+      "description": "Mentors provide both technical and non-technical support to the hackers throughout the hackathon.",
+    },
+    {
+      "title": "Started Freelancing",
+      "date": "2022",
+      "icon": "FaDollarSign",
+      "color": "yellow",
+      "description": "Mentors provide both technical and non-technical support to the hackers throughout the hackathon.",
+    },  
```

```suggestion
    }
```
Please remove this comma